pipelines/openjdk10_openj9_nightly_pipeline.groovy

```diff
@@ -2,9 +2,9 @@ println "building ${JDK_VERSION}"
 
 def buildPlatforms = ['Linux', 'zLinux', 'ppc64le', 'AIX']
 def buildMaps = [:]
-buildMaps['Linux'] = [test:false, ArchOSs:'x86-64_linux']
+buildMaps['Linux'] = [test:true, ArchOSs:'x86-64_linux']
 buildMaps['zLinux'] = [test:false, ArchOSs:'s390x_linux']
-buildMaps['ppc64le'] = [test:false, ArchOSs:'ppc64le_linux']
+buildMaps['ppc64le'] = [test:true, ArchOSs:'ppc64le_linux']
 buildMaps['AIX'] = [test:false, ArchOSs:'ppc64_aix']
 def typeTests = ['openjdktest', 'systemtest']
 
```

Comment:
openjdktest is not ready due to:
Exception in thread "main" java.lang.UnsatisfiedLinkError: sun/hotspot/WhiteBox.registerNatives()V
in openjdk10-openj9 openjdktest build https://ci.adoptopenjdk.net/view/work%20in%20progress/job/openjdk10_j9_openjdktest_x86-64_linux/2/console

Wouldn't suggest to enable it.

Reply:
OK. Will wait.

Reply:
@Mesbah-Alam you can enable systemtest first. Just comment out  def typeTests = ['openjdktest', 'systemtest'] and add  def typeTests = ['systemtest']

Reply:
Done! Please review.